x-pack/plugin/esql/build.gradle

```diff
@@ -40,23 +40,8 @@ dependencies {
   internalClusterTestImplementation project(":modules:mapper-extras")
 }
 
-/*
- * IntelliJ will always put the java files generated by the annotation processor
- * into src/main/java/generated so we make gradle play along. This block makes
- * it put the generated files into the same spot and the next block stops it from
- * trying to compile the generated files in the regular compile - it'll regenerate
- * them and *then* compile them.
- */
 tasks.named("compileJava").configure {
-  options.compilerArgs.addAll(["-s", "${projectDir}/src/main/java/generated"])
```

Comment:
It's now making them in `src/main/generated-src/generated` and I can't move it. It looks like this isn't configurable in intellij. But the `compute` subproject does the right thing and intellij generates the classes to `src/main/generated`. I have no idea what's up. But this works around it.

If any expert in gradle has time to understand this, great! You could fix this a lot better than I can.

Reply:
Ok. This seems to work.